server/routes/validators/induction/workInterestRolesFormValidator.ts

```diff
@@ -0,0 +1,52 @@
+import type { WorkInterestRolesForm } from 'inductionForms'
```

Comment:
New validator for `WorkInterestRolesForm` - this didnt exist before.
Now that we are validating it, this is why it has to live on the session (see above)